scripts/linking.py

```diff
@@ -245,6 +245,7 @@ def load_tfidf_ann_index(model_path: str):
     # It makes the query slow without significant gain in recall.
     efS = 1000
     tfidf_vectorizer_path = f'{model_path}/tfidf_vectorizer.joblib'
+    linking_classifier_path = f'{model_path}/linking_classifier.joblib'
```

Comment:
This is conceptually separate from the candidate generation step, as we might train different classifiers. Could you separate the loading and saving of it from this function and make it optional in the top level script?

Reply:
done here https://github.com/allenai/scispacy/pull/106